modules/apps/29-fee/keeper/escrow.go

```diff
@@ -165,6 +166,7 @@ func (k Keeper) distributeFee(ctx sdk.Context, receiver, refundAccAddress sdk.Ac
 		// then attempt to refund the fee to the original sender
 		err := k.bankKeeper.SendCoinsFromModuleToAccount(cacheCtx, types.ModuleName, refundAccAddress, fee)
 		if err != nil {
+			k.Logger(ctx).Error("Error refunding fee to the original sender", "Refund Address:", refundAccAddress, "Fee:", fee)
```

Comment:
Sorry, this is super nit, but I think in other places where we log we do not use capital letters or colons...
```suggestion
			k.Logger(ctx).Error("error refunding fee to the original sender", "refund address", refundAccAddress, "fee", fee)
```